bibliopixel/led/__init__.py

```diff
@@ -1,20 +1,27 @@
-from . circle import LEDCircle
-from . matrix import LEDMatrix
-from . cube import LEDCube
-from . pov import LEDPOV
-from . strip import LEDStrip
+from . circle import Circle
+from . matrix import Matrix
+from . cube import Cube
+from . pov import POV
+from . strip import Strip
 from .. layout import Rotation
 from .. layout.matrix import gen_matrix
 from .. layout.circle import gen_circle
 from .. layout.cube import gen_cube
 
+# These are DEPRECATED
```

Comment:
Nah.  It's just a comment for the reader.  Everything that was defined before, is still defined now - however, the old names are now just synonyms for the new, shorter names.